log-example/src/main/resources/log4j.properties

```diff
@@ -1,8 +1,16 @@
-log4j.rootLogger=info, stdout, http
+log4j.rootLogger=info, stdout, http, file
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%5p [%t] %c - %m%n
 
+log4j.appender.file=org.apache.log4j.FileAppender
+log4j.appender.file.File=./logs/StandAloneAppLog.out
```

Comment:
nit use extension `.log`.

Reply:
Is the log dir automatically created?

Reply:
@afoerster Yes the log dir is automatically created